assets/javascripts/discourse/components/chat-live-pane.js

```diff
@@ -767,10 +767,10 @@ export default Component.extend({
     return Promise.resolve();
   },
 
-  _onSendError(stagedId) {
+  _onSendError(stagedId, error) {
     const stagedMessage = this.messageLookup[`staged-${stagedId}`];
     if (stagedMessage) {
-      stagedMessage.set("error", true);
+      stagedMessage.set("error", error.jqXHR.responseJSON.errors[0]);
```

Comment:
I know it's ugly to do this, but I don't want to use popupError, and want just to text of the error 🤷